### PR TITLE
Added Hebrew ordinal numbers

### PR DESCRIPTION
**Pull request by nsantacruz**

### CODE REVIEW

File: num2words/lang_HE.py [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+
+    words = []
+
+    chunks = list(splitby3(str(n)))
+    i = len(chunks)
+    for x in chunks:
+        i -= 1
+        n1, n2, n3 = get_digits(x)
+
+        # print str(n3) + str(n2) + str(n1)
```
Comment (hsoft) [inline num2words/lang_HE.py]: spurious?

File: out.txt [hunk 2]
```diff
@@ -0,0 +1,12 @@
+1 אחד
```
Comment (hsoft) [inline out.txt]: spurious?

**Comment by hsoft** [comment]
Thanks @nsantacruz ! As I write in the review, it looks good except with some contents which seems to have been mistakenly committed. If you remove them, then it would be good to merge I think.

**Comment by nsantacruz** [comment]
No prob @hsoft. I only added in ordinal numbers because I'm familiar with that and the code is working for the application I needed. Hope it's helpful for others as well

**Comment by hsoft** [comment]
And what about the commented out `print`?

**Comment by nsantacruz** [comment]
That's a basic test. I'm not sure what the accepted practice is in the
repo, but I thought it would be nice to leave a basic test left in case
someone wants to update the code.

**Comment by hsoft** [comment]
Merged in https://github.com/savoirfairelinux/num2words/commit/182b82cf1ff47d588f50b1feb650e721626cbbd1 , thanks!

**Pull request closed without merging**